tezos/caip10.md

```diff
@@ -0,0 +1,55 @@
+---
+namespace-identifier: tezos-caip10
+title: Tezos Namespace - Addresses
+author: Qibing Li (@QibingLee)
+discussions-to: TBA
+status: Draft
+type: Standard
+created: 2022-12-06
+updated: 2022-12-06
+requires: ["CAIP-2", "CAIP-10"]
+---
+
+# CAIP-10
+
+_For context, see the [CAIP-10][] specification._
+
+## Rationale
+
+Tezos account is linked to a manager, which owns the public key. The hash of the public key outputs an address. Tezos supports the use of multiple public-key signature schemes, so the address prefix starts with `tz1` (Ed25519 curve), `tz2` (Secp256k1 curve), or `tz3` (P256 curve).
```

Comment:
is there a similar distinction (to eth/eip155) in tezos between EOA (externally owned accounts) and contract based accounts? 

if contract based account here, can the manager change?

may want to describe each separately, and be able to identity both contract based account and eoa/keypair

Reply:
The manager is not the contract-based account. Actually, everything starts with a random number, then users can select different curves to generate different <public-key, private-key> pairs (tz1, tz2, tz3). They are totally different accounts to manage different funds. One advantage is, holding the random number makes you to take back funds across there accounts.